client/src/components/home/Home.tsx

```diff
@@ -0,0 +1,141 @@
+              <StackItem>
+                <DefaultButton text="Book an appointment" styles={buttonStyle} iconProps={{ iconName: 'Calendar' }} />
+              </StackItem>
+              <StackItem>
+                <DefaultButton
+                  iconProps={{ iconName: 'Video' }}
+                  text="Start a call"
+                  split
+                  splitButtonAriaLabel="See 2 options"
+                  aria-roledescription="split button"
+                  menuProps={menuProps}
+                  styles={buttonStyle}
+                />
+              </StackItem>
+              <StackItem>
+                <DefaultButton text="Join from link" styles={buttonStyle} iconProps={{ iconName: 'Link' }} />
+              </StackItem>
+            </Stack>
+            <Text styles={font16pxStyle}>Frequently asked questions</Text>
+            <Stack styles={btnStackStyles}>
+              <Link tabIndex={0} data-testid="bookingsSetupLink" target="_blank">
+                <Stack horizontal disableShrink>
+                  <StackItem align="center" style={linkStyles}>
+                    How do I change my Microsoft Bookings page URL?
+                  </StackItem>
+                  <StackItem align="center" style={textDecorationNone}>
+                    <span style={newWindowIconWrapper}>
+                      <Icon iconName="OpenInNewWindow" />
+                    </span>
+                  </StackItem>
+                </Stack>
+              </Link>
+              <Link tabIndex={0} data-testid="bookingsSetupLink" target="_blank">
+                <Stack horizontal disableShrink>
+                  <StackItem align="center" style={linkStyles}>
+                    How do I switch from using Rooms (preview) to using Microsoft Bookings?
+                  </StackItem>
+                  <StackItem align="center" style={textDecorationNone}>
+                    <span style={newWindowIconWrapper}>
+                      <Icon iconName="OpenInNewWindow" />
+                    </span>
+                  </StackItem>
+                </Stack>
+              </Link>
+              <Link tabIndex={0} data-testid="bookingsSetupLink" target="_blank">
+                <Stack horizontal disableShrink>
+                  <StackItem align="center" style={linkStyles}>
+                    More frequently asked questions
+                  </StackItem>
+                  <StackItem align="center" style={textDecorationNone}>
+                    <span style={newWindowIconWrapper}>
+                      <Icon iconName="OpenInNewWindow" />
+                    </span>
+                  </StackItem>
+                </Stack>
+              </Link>
+            </Stack>
+            <Text styles={font16pxStyle}>Learn more about Azure Communication Services</Text>
+            <Link styles={font12pxSemiBoldStyle}>Azure Communication Services virtual appointments</Link>
+            <Text styles={font12pxWeight400Style}>
```

Comment:
We are seeing a pattern here. There is always a Link (header) and a Text (description). Let's extract this to a component where we can customize via props to avoid repetition of code:
```
<LearnMoreItem
    headerText={"Learn more about Azure Communication Services"}
    headerLink={aka.ms/.......}
    description={"This tutorial...."}
/>
```

Reply:
Thank you for the suggestion. Done!